src/shim/ResizeObserver.ts

```diff
@@ -33,7 +33,8 @@ export interface ResizeObserver {
 
 if (!has('build-elide')) {
 	if (!global.ResizeObserver) {
-		global.ResizeObserver = Resize;
+		// default is undefined when UMD module is used
+		global.ResizeObserver = Resize.default || Resize;
```

Comment:
I don't think you need this change, when doing `* as Resize` it works correctly in non-umd environments.

Reply:
Isn't `default` the same as a named export when doing a namespace import?

Reply:
I get this error when I test this with a Dojo Webpack build without changing this line:

`TypeError: Object is not a constructor (evaluating 'new a.default')`

Also `window.ResizeObserver` is set to an Object that has a `default` property.

Reply:
@dasa yup i was being silly and testing this in chrome which has native support 😂 . thanks!